src/univariate/continuous/normal.jl

```diff
@@ -74,22 +73,20 @@ kurtosis(d::Normal{T}) where {T<:Real} = zero(T)
 
 entropy(d::Normal) = (log2π + 1)/2 + log(d.σ)
 
-
 #### Evaluation
 
-@_delegate_statsfuns Normal norm μ σ
-
-gradlogpdf(d::Normal, x::Real) = (d.μ - x) / d.σ^2
-
-mgf(d::Normal, t::Real) = exp(t * d.μ + d.σ^2/2 * t^2)
-cf(d::Normal, t::Real) = exp(im * t * d.μ - d.σ^2/2 * t^2)
-
+logpdf(d::Normal, x::Real) = -(x - d.μ)^2 / 2 / d.σ^2 - log(2π * d.σ^2) / 2
+pdf(d::Normal, x::Real) = exp(logpdf(d, x))
+cdf(d::Normal, x::Real) = (1 + erf((x - d.μ) / d.σ / √2)) / 2
+logcdf(d::Normal, x::Real) =
+    erf((x - d.μ) / d.σ / √2) |>
+    (value -> value < -1 ? log1p(value) - log(2) : log((1 + value) / 2))
```

Comment:
This is a bit too smart for me, can we do it without `|>`?

Reply:
+1 yup, plus @ararslan is gonna block this PR with a `|>` :stuck_out_tongue: